LICENSE

```diff
@@ -1,339 +0,0 @@
-                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Like for README, please check whether removing this file will break anything ...